bread/layout/admin.py

```diff
@@ -165,6 +166,7 @@ class ReindexForm(forms.Form):
             ),
             style="margin-bottom: 1rem;",
         ),
+        tooltip.InteractiveTooltip("Test Label", "Test Body"),
         reindex_btn,
         hg.If(
             logmsg,
```

Comment:
Can we remove the changes in this file? If we want to have something for preview on the UI I recommend you add it on the view ```bread.views.admin.widgetpreview```.

Reply:
I forgot to remove that. Sorry.